### PR TITLE
Use github actions to do CI

### PR DESCRIPTION
**Pull request by typhoonzero**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        bash scripts/test/java.sh
+    - name: workflow mode ci
+      run: |
+        set -e
+        source build/env/bin/activate
+        docker tag sqlflow/sqlflow:mysql sqlflow:mysql
+        bash scripts/test/workflow.sh
+        docker rmi sqlflow:mysql
+        # bash scripts/travis/upload_codecov.sh
+  # push:
```
Comment (typhoonzero) [inline .github/workflows/main.yml:65]: Will finish these in the next PR.

File: docker/dev/find_fastest_resources.sh [hunk 2]
```diff
@@ -237,6 +237,7 @@ function choose_fastest_apt_source() {
     install_requirements_if_not
     echo "Find fastest APT mirror ..."
     find_fastest_apt_source > /etc/apt/sources.list
+    cat /etc/apt/sources.list
```
Comment (Yancey0623) [inline docker/dev/find_fastest_resources.sh]: This line is just used for debugging?

File: scripts/test/mysql.sh [hunk 3]
```diff
@@ -14,18 +14,18 @@
...
-# Wait for the creation of file /work/mysql-inited.  The entrypoint
-# of sqlflow:mysql should create this file on a bind mount of the host
-# filesystem.  So, the container running this script should also bind
-# mount the same host directory to /work.
-while read -r i; do if [ "$i" = "mysql-inited" ]; then break; fi; done \
-    < <(inotifywait  -e create,open --format '%f' --quiet /work --monitor)
+
+port=$(echo "$SQLFLOW_TEST_DB_MYSQL_ADDR" | awk -F":" '{print $2}')
+echo "Waiting mysql to launch on $port..."
+while ! nc -z localhost "$port"; do
```
Comment (Yancey0623) [inline scripts/test/mysql.sh]: I believed the MySQL container would write a file `mysql-inited` after loading the example datasets, so we can not do this change?
https://github.com/sql-machine-learning/sqlflow/blob/develop/docker/mysql/start.bash#L75

Comment (typhoonzero) [reply scripts/test/mysql.sh]: We start the MySQL container using docker, so the file `mysql-inited` is owned by root user, which can not be read by a normal Linux user.

File: scripts/test/prepare.sh [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    seaborn==0.9.0 \
+    scikit-learn==0.21.0 \
+    sklearn2pmml==0.56.0 \
+    jpmml-evaluator==0.3.1 \
+    PyUtilib==5.8.0 \
+    pyomo==5.6.9 \
+    mysqlclient==1.4.4 \
+    pytest
+
+git clone https://github.com/sql-machine-learning/models.git
```
Comment (sneaxiy) [inline scripts/test/prepare.sh:45]: Need to checkout `v0.0.4`?

File: go/cmd/sqlflowserver/e2e_common_cases.go [hunk 5]
```diff
@@ -651,6 +651,8 @@ func caseXGBoostSparseKeyValueColumn(t *testing.T) {
 
 	if testDBType == "maxcompute" {
 		dbName = caseDB
+	} else if testDBType == "hive" {
```
Comment (sneaxiy) [inline go/cmd/sqlflowserver/e2e_common_cases.go]: Why skip this ut?

**Review by shendiaomo — APPROVED** [review]

**Pull request merged**